.github/workflows/release.yml

```diff
@@ -47,15 +47,19 @@ jobs:
         awk "/# ${GITHUB_REF_NAME}/{flag=1; next} /^$/{flag=0} flag" CHANGELOG.md >REL.md
 
     - name: Build stage1
-      run: cargo zigbuild --release --target ${{ matrix.target }}
+      run: cargo build --release
 
     - name: Switch to linking with wild
       run: |
         mkdir -p "$HOME/.cargo"
         echo -e "[target.${{matrix.target}}]\nlinker = \"clang\"\nrustflags = [\"-Clink-arg=--ld-path=${PWD}/target/${{matrix.target}}/release/wild\"]\n" >>"$HOME/.cargo/config.toml"
 
-    - name: Build release binary
+    - name: Build release binary for gnu-libc
+      if: contains(matrix.target, 'gnu')
       run: cargo zigbuild --profile dist --target ${{ matrix.target }}
+    - name: Build release binary for musl-libc
```

Comment:
Would it work to add features to the matrix? e.g. `--features "${{ matrix.features }}"`. Then we wouldn't need separate conditional steps